src/xtools/xdb_tool/src/xdb_tool.cpp

```diff
@@ -20,15 +20,15 @@
 
 
 #define CONS_AUDITOR_COUNT 1
-#define CONS_VALIDATOR_COUNT 1
-#define NETWORK_ID 0
+#define CONS_VALIDATOR_COUNT 2
+#define NETWORK_ID 255
```

Comment:
why
CONS_AUDITOR_COUNT 1? 
CONS_VALIDATOR_COUNT 2?
NETWORK_ID 255?

only for local testnet?

Reply:
current adhere to local run.sh build_dev test (1cluster/2auditor